conf/post_deploy_actions.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# abort on any errors
+set -e
+
+# check that we are in the expected directory
+cd `dirname $0`/..
+
+# install sass locally
+mkdir -p "../gems"
+export GEM_HOME="$(cd ../gems && pwd -P)"
+echo $GEM_HOME
+export PATH="$GEM_HOME/bin:$PATH"
+
+if [ ! -f ../gems/bin/sass ]; then
+    gem install --no-ri --no-rdoc sass    # -v 3.2.14
```

Comment:
FixMyStreet is now using bundler to do this, so this should behave the same. And could it then be called from `install-as-user` perhaps?